### PR TITLE
README: Update meeting times

### PR DESCRIPTION
**Pull request by randomvariable**
**What this PR does / why we need it**:
Updates the meeting times to reflect reality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ This repository uses the Kubernetes bots.  See a full list of the commands [here
 
 ### Implementer office hours
 
-Maintainers hold office hours every week, with sessions open to all developers
-working on this project.
+Maintainers hold office hours every fortnight, with sessions open to al
```
Comment (detiber) [inline README.md]:
```suggestion
Maintainers hold office hours every fortnight, with sessions open to all
```

Comment (detiber) [reply README.md]: I'm not necessarily against the use of the word fortnight here, but I do wonder if it's not common enough terminology compared to `biweekly`

File: README.md [hunk 2]
```diff
@@ -67,10 +67,10 @@ This repository uses the Kubernetes bots.  See a full list of the commands [here
...
+developers working on this project.
 
-Office hours are hosted on a zoom video chat on Mondays
+Office hours are hosted on a zoom video chat every 2 Mondays
```
Comment (detiber) [inline README.md]:
```suggestion
Office hours are hosted on a zoom video chat every other Monday
```

File: README.md (hunk 1 above)
Comment (randomvariable) [reply README.md]: The British have been eliminated.

**Comment by detiber** [comment]
:joy: at the second commit message

/lgtm

**Pull request merged**